ssl/quic/quic_port.c

```diff
@@ -1536,6 +1597,9 @@ static void port_default_packet_handler(QUIC_URXE *e, void *arg,
      * server address validation, we may still get a token if we sent
      * a NEW_TOKEN frame during a prior connection, which we should still
      * validate here
+     *
+     * TODO: is it OK to keep qrx in this case? Are not we running into
```

Comment:
What do we plan to do this with `TODO`?

Reply:
thank you for the question. I re-think the matter over a lunch. I think packets with tokens earned by NEWTOKEN frame should be treated in the same way like packets with token issued via RETRY packet. will fix it.

Reply:
I agree that NEW_TOKEN tokens are no different from RETRY tokens and should be handled in exactly the same way (save for any control flow discrepancies in port_validate_token based on the is_retry boolean)